services/app/package.json

```diff
@@ -45,10 +45,12 @@
     "webpack-dev-server": "^4.6.0"
   },
   "dependencies": {
+    "bootstrap": "^5.1.3",
```

Comment:
Is bootstrap necessary?

Reply:
@jbartka is this needed?

Reply:
I tried removing the bootstrap from the .json file, and it seems to be working fine, so I guess we don't really need it. Besides, we are not importing this dependency into any of our directory files; thus, it would not make much sense to have it here anyways.